python/langsmith/run_trees.py

```diff
@@ -855,7 +880,22 @@ def from_header(cls, header_value: Optional[str]) -> _Baggage:
                     project_name = urllib.parse.unquote(value)
                 elif key == LANGSMITH_REPLICAS:
                     replicas_data = json.loads(urllib.parse.unquote(value))
-                    replicas = [(str(proj), updates) for proj, updates in replicas_data]
+                    parsed_replicas: list[Replica] = []
+                    for replica_item in replicas_data:
+                        if isinstance(replica_item, list) and len(replica_item) == 2:
+                            # Legacy format: [project_name, updates]
```

Comment:
Should we still load as a dict ? Feels nice to have consistent format once it's actually loaded